### PR TITLE
Update rest presentation

### PR DESCRIPTION
**Pull request by augustd**
Replaced link to Presentation "Bypassing Web Authentication and Authorization with HTTP Verb Tampering" with local PDF file in assets directory. 

```
This PR covers issue #80.
```

Thanks you again for your contribution :smiley:

### CODE REVIEW

**Comment by righettod** [comment]
Hi, thanks you very much for the PR. It is possible to rename the pdf file
to match this convention:
[CHEAT_CHEET_MARKDOWN_FILE_NAME]_[IDENTIFIER].[EXTENSION]
? The name will become REST_Security_Cheat_Sheet_Pdfnamecamelcase.pdf
tha ks you
very much in advance and thanks again a lot for the PR.

**Comment by righettod** [comment]
Hi, 

Thanks you very much for the PR. It is possible to rename the pdf file to match this convention?

`[CHEAT_CHEET_MARKDOWN_FILE_NAME]_[IDENTIFIER].[EXTENSION]`

The name will become `REST_Security_Cheat_Sheet_Bypassing_VBAAC_with_HTTP_Verb_Tampering.pdf`

Thanks you very much in advance and thanks again a lot for the PR.

File: cheatsheets/REST_Security_Cheat_Sheet.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ API keys can reduce the impact of denial-of-service attacks. However, when they
 - Reject all requests not matching the whitelist with HTTP response code `405 Method not allowed`.
 - Make sure the caller is authorised to use the incoming HTTP method on the resource collection, action, and record
 
-In Java EE in particular, this can be difficult to implement properly. See [Bypassing Web Authentication and Authorization with HTTP Verb Tampering](http://www.aspectsecurity.com/research-presentations/bypassing-vbaac-with-http-verb-tampering) for an explanation of this common misconfiguration.
+In Java EE in particular, this can be difficult to implement properly. See [Bypassing Web Authentication and Authorization with HTTP Verb Tampering](../assets/Bypassing_VBAAC_with_HTTP_Verb_Tampering.pdf) for an explanation of this common misconfiguration.
```
Comment (righettod) [inline cheatsheets/REST_Security_Cheat_Sheet.md]: Hi, 

Thanks you very much for the PR. It is possible to rename the pdf file to match this convention?

`[CHEAT_CHEET_MARKDOWN_FILE_NAME]_[IDENTIFIER].[EXTENSION]`

The name will become `REST_Security_Cheat_Sheet_Bypassing_VBAAC_with_HTTP_Verb_Tampering.pdf`

Thanks you very much in advance and thanks again a lot for the PR.

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,4 @@ TOC.md
 node_modules/
 site/
 generated/
+.DS_Store
```
Comment (mackowski) [inline .gitignore]: please delete this file

File: cheatsheets/REST_Security_Cheat_Sheet.md (hunk 1 above)
Comment (righettod) [reply cheatsheets/REST_Security_Cheat_Sheet.md]: I will handle the renaming 😃

**Pull request merged**